include/RemediationRoleSaver.h

```diff
@@ -22,52 +22,143 @@
 #ifndef SCAP_WORKBENCH_REMEDIATION_ROLE_SAVER_H_
 #define SCAP_WORKBENCH_REMEDIATION_ROLE_SAVER_H_
 
-#include "ForwardDecls.h"
```

Comment:
in Workbench we include ForwardDecls in every file as the first include on purpose. You never know what's going to be added to that file.